tempdb.go

```diff
@@ -76,6 +84,11 @@ func (sl *DB) PrintStats() string {
 }
 
 func New(args ...any) (walletdb.DB, error) {
+	if Logger == nil {
+		h := slog.NewTextHandler(io.Discard, &slog.HandlerOptions{Level:slog.LevelError})
```

Comment:
options should be `nil` here as everything seems to be ignored when using `io.Discard`.

Reply:
fixed in https://github.com/linden/tempdb/pull/1/commits/d0180d708cdb229b7186967a1691b0df787bebec.